src/components/items/Stars.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import full from './stars/full.svg';
+import half from './stars/half.svg';
+import empty from './stars/empty.svg';
+
+const Stars = (props) => {
+  const { rating } = props;
+
+  const stars = [];
+  // a loop to push stars to the above array based on rating
+  for (let i = stars.length; i < 5; i++) {
```

Comment:
Really good job with this!!

Reply:
Thanks, this makes me happy. I was afraid I wasn't using enough finesse with a for-loop, so I'm glad this is an acceptable solution.